packages/office-ui-fabric-react/src/components/Link/Link.scss

```diff
@@ -42,6 +41,11 @@
   &:active {
     color: $ms-color-themePrimary;
   }
+
+  &:focus {
```

Comment:
Once this is in a new `focus-outline()` mixin, use it only on `a.ms-Link` and continue using `focus-border()` for `button.ms-Link`.